packages/prosess-vedtak/src/components/VedtakForm.tsx

```diff
@@ -263,7 +263,10 @@ export const VedtakForm: React.FC<Props> = ({
           !kanHaAutomatiskVedtaksbrev(tilgjengeligeVedtaksbrev) &&
           !harMellomLagretMedIngenBrev(dokumentdata, vedtakVarsel)) ||
         (kanHaManueltFritekstbrev(tilgjengeligeVedtaksbrev) &&
-          harSattDokumentdataType(dokumentdata, vedtakVarsel, vedtaksbrevtype.MANUELL)),
+          harSattDokumentdataType(dokumentdata, vedtakVarsel, vedtaksbrevtype.MANUELL)) ||
+        (kanHaManueltFritekstbrev(tilgjengeligeVedtaksbrev) &&
```

Comment:
jeg dupliserte fritekstV1 sjekken for manuelt brev. MEN @vebnor  eller @thomashwi  hva med 
```(harFritekstILokalState && kanHaFritekstbrevV1(tilgjengeligeVedtaksbrev))```
bør den også dupliseres med 
```(harFritekstILokalState && kanHaManueltFritekstbrev(tilgjengeligeVedtaksbrev))```

Reply:
@qtips  Lokal state blir ikke relevant med manuelle brev slik vi har bygget det opp. I og med at man får opp editor i modal og lagrer for å lukke.